website/docs/r/api_management_cache.html.markdown

```diff
@@ -0,0 +1,93 @@
+---
+subcategory: "API Management"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_api_management_cache"
+description: |-
+  Manages a API Management Cache.
+---
+
+# azurerm_api_management_cache
+
+Manages a API Management Cache.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_api_management" "example" {
+  name                = "example-apim"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+  publisher_name      = "pub1"
+  publisher_email     = "pub1@email.com"
+  sku_name            = "Consumption_0"
+}
+
+resource "azurerm_redis_cache" "example" {
+  name                = "example-cache"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+  capacity            = 1
+  family              = "C"
+  sku_name            = "Basic"
+  enable_non_ssl_port = false
+  minimum_tls_version = "1.2"
+
+  redis_configuration {
+  }
+}
+
+resource "azurerm_api_management_cache" "example" {
+  name              = "example-Apim-Cache"
+  api_management_id = azurerm_api_management.example.id
+  connection_string = azurerm_redis_cache.example.primary_connection_string
+  description       = "Redis cache instances"
+  redis_cache_id    = azurerm_redis_cache.example.id
+  use_from_location = "East Us"
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name which should be used for this API Management Cache. Changing this forces a new API Management Cache to be created.
+
+* `api_management_id` - (Required) The resource ID of the Api Management Service from which to create this external cache. Changing this forces a new API Management Cache to be created.
+
+* `connection_string` - (Required) The connection string to the Cache for Redis.
+
+---
+
+* `description` - (Optional) The description of the API Management Cache.
+
+* `redis_cache_id` - (Optional) The resource ID of the Cache for Redis.
+
+* `use_from_location` - (Optional) The location where to use cache from. Possible values are `default` and valid Azure regions. Defaults to `default`.
```

Comment:
this is not the best name, what about
```suggestion
* `cache_location` - (Optional) The location where to use cache from. Possible values are `default` and valid Azure regions. Defaults to `default`.
```

also how is this different from the redis cache location